tests/integration/_experimental/object_detection/test_evaluator_multiclass_test_sample_metrics.py

```diff
@@ -0,0 +1,2672 @@
+                TP=[
+                    ScoredLabeledBoundingBox(top_left=(23.0, 23.0), bottom_right=(24.0, 24.0), label="e", score=0.8),
+                    ScoredLabeledBoundingBox(top_left=(25.0, 25.0), bottom_right=(26.0, 26.0), label="e", score=0.7),
+                    ScoredLabeledBoundingBox(top_left=(27.0, 27.0), bottom_right=(28.0, 28.0), label="e", score=0.6),
+                    ScoredLabeledBoundingBox(top_left=(31.0, 31.0), bottom_right=(32.0, 32.0), label="e", score=0.1),
+                ],
+                FP=[
+                    ScoredLabeledBoundingBox(top_left=(21.0, 21.0), bottom_right=(22.0, 22.0), label="b", score=0.9),
+                    ScoredLabeledBoundingBox(top_left=(1.0, 1.0), bottom_right=(2.0, 2.0), label="b", score=0.7),
+                    ScoredLabeledBoundingBox(top_left=(3.0, 3.0), bottom_right=(4.0, 4.0), label="b", score=0.6),
+                ],
+                FN=[
+                    LabeledBoundingBox(top_left=(21.0, 21.0), bottom_right=(22.0, 22.0), label="e"),
+                    LabeledBoundingBox(top_left=(33.0, 33.0), bottom_right=(34.0, 34.0), label="e"),
+                    LabeledBoundingBox(top_left=(35.0, 35.0), bottom_right=(36.0, 36.0), label="e"),
+                    LabeledBoundingBox(top_left=(37.0, 37.0), bottom_right=(38.0, 38.0), label="e"),
+                    LabeledBoundingBox(top_left=(41.0, 41.0), bottom_right=(42.0, 42.0), label="e"),
+                    LabeledBoundingBox(top_left=(43.0, 43.0), bottom_right=(44.0, 44.0), label="e"),
+                ],
+                Confused=[
+                    ScoredLabeledBoundingBox(top_left=(21.0, 21.0), bottom_right=(22.0, 22.0), label="b", score=0.9),
+                ],
+                count_TP=4,
+                count_FP=3,
+                count_FN=6,
+                count_Confused=1,
+                has_TP=True,
+                has_FP=True,
+                has_FN=True,
+                has_Confused=True,
+                max_confidence_above_t=0.9,
+                min_confidence_above_t=0.1,
+                thresholds=[ScoredClassificationLabel(label="e", score=0.1)],
+            ),
+        ),
+    ],
+}
+
+
+@pytest.mark.parametrize(
+    "config_name, test_name",
+    TEST_PARAMS,
+)
+def test__prebuilt__object__detection__multiclass__compute__test__sample__metrics(
+    config_name: str,
+    test_name: str,
+) -> None:
+    config = TEST_CONFIGURATIONS[config_name]
+    eval = ObjectDetectionEvaluator(configurations=[config])
+    result = eval.compute_test_sample_metrics(
+        test_case=TEST_CASE,
+        inferences=TEST_DATA[test_name],
+        configuration=config,
+    )
+    print(config_name, test_name)
+    print(result)
+    assert EXPECTED_COMPUTE_TEST_SAMPLE_METRICS[config_name, test_name] == result
+
+
+def test__prebuilt__object__detection__multiclass__compute__test__sample__metrics__all() -> None:
+    config_name = "Threshold: Fixed(0.5) by class, IoU: 0.5, confidence ≥ 0.0"
```

Comment:
These names are going to be quite brittle. Is there a reason we're not just using the config object itself, rather than its stringified display name?